### DISCUSSION TITLE
Issues with i20221117 on new SD card

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1103] [Fixed Issues]
Since flashing a new SD card to  i20221117 the Zoom setting for the RPi Official 7in Screen seems to have stopped working.  I usually have it set to 70% but no matter what setting I use now nothing seems to affect the actual screen size.

Also, before reflashing a new SD card I took a backup of my original configuration to get a backup.gz file.  I added this backup file to the Root partition of the new card along with 'ssh' and 'wifi' files which is my normal means of provisioning a new card.  On booting from the new card the 'ssh' and 'wifi' files were processed but the backup wasn't.  I then tried a Restore from the System page using a copy of the original backup file which seem to work ok so I am not sure there is anything wrong with the backup.gz file.
Regards

Chris

**Comment by Vseason** [comment]
Can confirm zoom functionality appears broken in my installation of i20221117 as well.

**Comment by ceejayemm** [comment]
Update 20221123 fixes Zoom issue.  Just backup/restore issue now outstanding.

Thanks

Chris

**Comment by Vseason** [comment]
Zoom is apparently broken again in 20230123 (fresh install for testing purposes). The “+” and “-“ buttons are unresponsive, and the text field does not generate a keyboard prompt for input.
